test/kwtool/crawlers/schemas/keyword_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule Kwtool.Crawlers.Schemas.UserTest do
+  use Kwtool.DataCase, async: true
+
+  alias Kwtool.Crawlers.Schemas.Keyword
+
+  describe "changeset/2" do
+    test "returns valid keyword changeset when given valid attributes" do
+      keyword_params = %{phrase: "elixir", status: "added", user_id: 1}
+      changeset = Keyword.changeset(keyword_params)
+
+      assert changeset.valid?
+      assert changeset.changes.phrase == "elixir"
+      assert changeset.changes.status == "added"
+      assert changeset.changes.user_id == 1
+    end
+
+    test "sets default status added when given no status attribute in advance" do
+      keyword_params = %{phrase: "elixir", user_id: 1}
+      changeset = Keyword.changeset(keyword_params)
+
+      assert changeset.valid?
+      assert changeset.changes.phrase == "elixir"
+      assert changeset.changes.status == "added"
+      assert changeset.changes.user_id == 1
+    end
+
+    test "returns the updated status name when given the status" do
+      keyword_params = %{phrase: "elixir", user_id: 1, status: "in_process"}
+      changeset = Keyword.changeset(%Kwtool.Crawlers.Schemas.Keyword{}, keyword_params)
+
+      assert changeset.valid?
+      assert changeset.changes.phrase == "elixir"
+      assert changeset.changes.status == "in_process"
+      assert changeset.changes.user_id == 1
+    end
```

Comment:
We can also test an assoc constraint in changeset but we need to insert the record first

```elixir
keyword_params = %{phrase: "elixir", user_id: 1, status: "in_process"}

{:error, changeset} = 
  %Keyword{}
  |> Keyword.changeset(keyword_params)
  |> Repo.insert()

refute changeset.valid?
assert errors_on(changeset) == %{"user" => ["does not exist"]}
```